Imagine/Data/DataManager.php

```diff
@@ -137,7 +137,7 @@ public function find($filter, $path)
             throw new \LogicException(sprintf('The mime type of image %s was not guessed.', $path));
         }
 
-        if (0 !== mb_strpos($binary->getMimeType(), 'image/')) {
+        if (0 !== mb_strpos($binary->getMimeType(), 'image/') && $binary->getMimeType() !== 'application/pdf') {
             throw new \LogicException(sprintf('The mime type of image %s must be image/xxx got %s.', $path, $binary->getMimeType()));
```

Comment:
The error message should be modified to reflect your change:

```
The mime type of image %s must be image/xxx or application/pdf, got %s.
```

Reply:
Fixed